cmd/ocm/create/cluster/cmd.go

```diff
@@ -533,19 +558,115 @@ func promptName(argv []string) error {
 	return fmt.Errorf("A cluster name must be specified")
 }
 
-func promptBYOVPC(fs *pflag.FlagSet, connection *sdk.Connection, cmd *cobra.Command) error {
+func shouldPromptForClusterWideProxy(proxy c.ClusterWideProxy, ccs c.CCS, interactive bool) bool {
```

Comment:
we can just change it to  promptForClusterWideProxy

Reply:
I rather leave it as is, since the function returns boolean. The function answers the question - "should we prompt the user with cluster-wide proxy option"